### ISSUE TITLE
Need to have a research method

### ISSUE THREAD

**Issue by dnhandy** [ByteSyncSolutions/Unraveler#1]
Our final paper ought to have a method, and if we come up with it ahead of time, I think it'll help focus the work so we're not collecting data we don't care about.

**Comment by dnhandy** [comment]
I'm thinking something like this:

1) Randomly generate several initial sets of weights (maybe 3-5, since it takes a while to run). I think we need to do this and average the results so we're not just learning behavior that occurs because we're near a certain local optimum. This will hopefully give us data that is slightly independent of the initial weights.

2) For each set of weights from step 1, initialize the network to those weights, then run the network2 SDG with whatever regularization method is already there (whether it's L1 or L2, but we probably ought to document that in our paper) and the cross-entropy cost function for each variation we're testing:
- Statically at max_regularization
- Statically at min_regularization
- Once per decay function (we may just have one or try a couple, such as linear decay and exponential decay), going from max_regularization to min_regularization

3) For each run in step 2, gather the accuracy on the training data and the test data. I think these are the only two pieces of data we care about, aren't they?

**Comment by WebCandyLLC** [comment]
So all of our tests look like we are using DefaultWeightInitializer and used the QuadraticCost function. I want to take a vote and see, do you guys think we should do another set but with CrossEntropy and keep DefaultWeightInitializer, and/or expand to LargeWeightInitializer? I know we are coming close to the 11th hour so may not have the time then...

**Comment by dnhandy** [comment]
I don't think so. A good experiment seeks to hold everything constant
except for the one variable you want to analyze. Think let's us focus just
on the regularization. Besides, I'm almost done with a draft of the paper.

**Comment by WebCandyLLC** [comment]
Sounds good to me :) well done Dave with the results!
